PaddleCV/image_classification/utils/utility.py

```diff
@@ -115,6 +117,14 @@ def parse_args():
     add_arg('reader_thread',            int,    8,                      "The number of multi thread reader")
     add_arg('reader_buf_size',          int,    2048,                   "The buf size of multi thread reader")
     add_arg('interpolation',            int,    None,                   "The interpolation mode")
+    add_arg('use_ema',                  bool,   False,                  "Whether to use ExponentialMovingAverage.")
+    add_arg('use_aa',                   bool,   False,                  "Whether to use auto augment")
+    add_arg('ema_decay',                float,  0.9999,                 "The value of ema decay rate")
+    add_arg('warm_up_epochs',           float,  5.0,                    "The value of warm up epochs")
```

Comment:
int?

Reply:
warm up epochs can be float, if do not want to warm up one epoch, like in yolo v3.